pkg/ocm/accounts_mgmt.go

```diff
@@ -103,6 +105,30 @@ func (a accountsMgmt) UpdateSubscription(ctx context.Context, subscriptionID, op
 	return responseVal, nil
 }
 
+func (a accountsMgmt) UpdateSubscriptionDisplayName(ctx context.Context, subscriptionID strfmt.UUID, displayName string) (*amgmtv1.Subscription, error) {
+	defer commonutils.MeasureOperation("OCM-UpdateSubscriptionDisplayName", a.client.log, a.client.metricsApi)()
+
+	// create the request
+	sub, err := amgmtv1.NewSubscription().DisplayName(displayName).Build()
+	if err != nil {
+		a.client.logger.Error(ctx, "Failed to create subscription request. Error: %v", err)
+		return nil, err
+	}
+
+	// send the request
```

Comment:
not crucial for this PR, but these 'send the request' lines look similar to lines 94-105, so probably worth extracting to an helper.

Reply:
Agree, will fix.